snow/engine/common/bootstrapper.go

```diff
@@ -162,22 +158,9 @@ func (b *bootstrapper) Startup(ctx context.Context) error {
 	return b.sendMessagesOrFinish(ctx)
 }
 
-func (b *bootstrapper) Restart(ctx context.Context, reset bool) error {
```

Comment:
`Restart(ctx, false)` is replaced with `Startup(ctx)` and `Restart(ctx, false)` is replaced with `Restart(ctx)`